lib/CL/devices/cuda/pocl-ptx-gen.cc

```diff
@@ -405,49 +409,42 @@ void fixPrintF(llvm::Module *Module) {
   VPrintF->eraseFromParent();
 }
 
-// TODO broken, replaces in whole module not just  1 function
-// Replace all load users of a scalar global variable with new value.
-#if 0
-static void replaceScalarGlobalVar(llvm::Module *Module, const char *Name,
-                                   llvm::Value *NewValue) {
+// Since all helper function are inlined, global variables only need to be replaced by arguments in current function
```

Comment:
Can you expand the comment a bit here: What does the function do and why? Please use the LLVM doxygen commenting style.